pulseworker/main.py

```diff
@@ -0,0 +1,116 @@
+import datetime
+import functools
+import logging
+import os
+import sys
+import json
+
+import requests
+from kinto_http import cli_utils
+from mozillapulse.consumers import NormalizedBuildConsumer
+
+
+DEFAULT_SERVER = "https://kinto-ota.dev.mozaws.net/v1"
+DEFAULT_BUCKET = "build-hub"
+DEFAULT_COLLECTION = "builds"
+PULSE_TOPIC = "build.#"
+PULSEGUARDIAN_USER = os.getenv("PULSEGUARDIAN_USER", "")
+PULSEGUARDIAN_PASSWORD = os.getenv("PULSEGUARDIAN_PASSWORD", "")
+
+
+logger = logging.getLogger(__name__)
+
+
+def epoch2iso(timestamp):
+    dt = datetime.datetime.utcfromtimestamp(timestamp)
+    # XXX: Python 3
+    if sys.version_info >= (3, 2):
+        dt = dt.replace(tzinfo=datetime.timezone.utc)
+    return dt.isoformat()
```

Comment:
Personally, I find this kind of code very scary. Could we use `arrow` or something instead?